### PR TITLE
Add debug utility

### PR DESCRIPTION
**Pull request by harshjv**
- [x] Add logging utility to `Client`
- [x] Log jsonrpc, metamask & ledger requests & response

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,11 +67,14 @@
     "bs58": "^4.0.1",
     "buffer-to-uint8array": "^1.1.0",
     "coinselect": "^3.1.11",
+    "console.history": "^1.5.0",
```
Comment (grandsmarquis) [inline package.json]: Is it use somewhere? You seems to have implemented your own?

File: package.json [hunk 2]
```diff
@@ -67,11 +67,14 @@
...
     "crypto-hashing": "^1.0.0",
+    "debug": "^4.1.1",
     "dsn-parser": "^1.0.1",
     "ethereumjs-tx": "^1.3.7",
     "json-bigint": "^0.3.0",
     "lodash": "^4.17.11",
+    "regenerator-runtime": "^0.13.2",
```
Comment (grandsmarquis) [inline package.json]: It also doesn't seems to be used

**Review by monokh — APPROVED** [review]
LGTM, minor comment

**Review by grandsmarquis — APPROVED** [review]
LGTM (just the 2 small comments)

**Pull request merged**